package.json

```diff
@@ -68,5 +69,6 @@
   "devDependencies": {
     "vite": "^5.2.7",
     "vite-plugin-dts": "^3.8.1"
```

Comment:
I just wonder why did I add this as dev dependency. Maybe it's time to clean it up and make the package totally dependency-free?

Reply:
yes, I think it can go away since it is not used in the root. you might want to consider upgrading all other dependencies as well, at least on dev.

Reply:
I'm going to change the tooling, so I will approve the PR and do it ina separate one

Reply:
Cool thanks for merging. If you need some inspiration I have a template for monorepo libs: https://github.com/pizzajsdev/template-monorepo-lib  , and https://github.com/pizzajsdev/pizza is a real example. Even though I use pnpm and tsup there.

Reply:
Anyway I am working on the yarn workspaces PR, and I am also adding a github workflow to make sure the builds dont break.